app-misc/piper/piper-0.5.1-r2.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{8..9} )
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
So python 3.10 doesn't work seem to work:

    # ~ $ piper 
    Traceback (most recent call last):
      File "/usr/bin/piper", line 36, in <module>
        from piper.application import Application
    ModuleNotFoundError: No module named 'piper'

Reply:
This looks like it is broken in meson:

    Program python3 (lxml, evdev, cairo, gi) found: YES (/usr/bin/python3.9) modules: lxml, evdev, cairo, gi
    Program python3 found: YES (/var/tmp/portage/app-misc/piper-0.6/temp/python3.10/bin/python3)

We maybe able to work around the problem with [0.62](https://github.com/mesonbuild/meson/pull/9788/files#diff-459a5de51a78f1bdaaa5b3032033df38a438d0c69fc66180d4a54a99c7e50714) but I am out of ideas on how to fix this for the current version.

Reply:
Never mind, I'm an idiot.  This works just fine, I built all the dependencies with python 3_10 and forgot to rebuild meson.

Reply:
don't worry, it happens to all :)